src/Aspire.Hosting.Azure/api/Aspire.Hosting.Azure.cs

```diff
@@ -226,7 +261,27 @@ internal AzureResourceInfrastructure() : base(default!) { }
         public AzureProvisioningResource AspireResource { get { throw null; } }
     }
 
-    public sealed partial class BicepOutputReference : ApplicationModel.IManifestExpressionProvider, ApplicationModel.IValueProvider, ApplicationModel.IValueWithReferences
+    public static partial class AzureUserAssignedIdentityExtensions
+    {
+        public static ApplicationModel.IResourceBuilder<AzureUserAssignedIdentityResource> AddAzureUserAssignedIdentity(this IDistributedApplicationBuilder builder, string name) { throw null; }
+    }
+
+    public sealed partial class AzureUserAssignedIdentityResource : AzureProvisioningResource, IAppIdentityResource
```

Comment:
Should it be named `AzureUserAssignedManagedIdentityResource`?

Reply:
Azure.Provisioning calls it `UserAssignedIdentity`, so I don't think "Managed" is necessary.

FYI - @tg-msft